requirements-dev.txt

```diff
@@ -0,0 +1,3 @@
+-r test-requirements.txt
```

Comment:
@jasonamyers can you explain (in more detail) why this should be in here? I thought we were just trying to get to an environment where it is possible to run the tests (with tox)?

Sorry for belaboring this, it is starting to go from hilarious to depressing, but if we're going to change it, I'd like to understand it and get it right.

Reply:
It depends on how you want the devs to work. Is tox the source of testing truth? if so, then we can dismiss this line. If not, then this will give them a virtualenv with all the tools in it.

Reply:
> Is tox the source of testing truth?

In gabbi itself, yes.

Reply:
for a developer when their setup their development environment what do you want them to have in it?

Reply:
That is the question I was trying to get an answer for earlier; whether there is anything beyond tox that's required. So I guess I'll just try it. One moment... tox, git, and an editor.